app/components/TabContents/TabContents.tsx

```diff
@@ -54,7 +54,8 @@ export class TabContents extends Component<{}, {}> {
         }
 
         const tabComponents = tabs.map( ( tab, i ) => {
-            const isActiveTab = tab.tabId === activeTabId;
+            console.log('shouldToggleDevTools', tab.tabId, tab.ui.shouldToggleDevTools);
+            const isActiveTab = tab.tabId === activeTabId ? true : false;
```

Comment:
lets drop the log, and I think the first version of this check was more terse / clearer,